devito/operator/operator.py

```diff
@@ -165,6 +168,10 @@ def __new__(cls, expressions, **kwargs):
     def _normalize_kwargs(cls, **kwargs):
         return kwargs
 
+    @classmethod
+    def _symbol_registry(cls):
+        return None
```

Comment:
Not an empty one `SymbolRegistry()`?

Reply:
On purpose None because this is an abstract Operator and not all possible concrete classes may need one. Our subclasses in `core` do, though